gallery/src/guidelines/FontSize/FontSizePage.tsx

```diff
@@ -45,21 +45,23 @@ const FontSizePage = () =>
 
     <section>
       <h3 className="nx-h3">Vocabulary</h3>
-      <dl className="nx-list nx-list--definition-list">
-        <dt className="nx-list__item nx-list__item--label">Glyph</dt>
-        <dd className="nx-list__item">The visible shape of a character</dd>
-        <dt className="nx-list__item nx-list__item--label">Baseline</dt>
-        <dd className="nx-list__item">
-          When writing or printing characters, the baseline is the <q>bottom</q> of standard characters. For instance,
-          when writing by hand on ruled paper, the rule line on which you are writing is the baseline. By default,
-          the characters in a sequence of text all have their baselines aligned.
-        </dd>
-        <dt className="nx-list__item nx-list__item--label">Descender</dt>
-        <dd className="nx-list__item">
-          Many glyphs, such as <q>q</q>, contain parts that drop below the baseline. Fonts and line layouts
-          must take this into account
-        </dd>
-      </dl>
+      <ul className="nx-list">
+        <li className="nx-list__item">
+          <span className="nx-list__text"><strong>Glyph:</strong> The visible shape of a character.</span></li>
```

Comment:
I feel like this looks pretty weird.  I wonder how it would be with the term as the `nx-list__text` and the definition as the subtext.

Reply:
> I wonder how it would be with the term as the nx-list__text and the definition as the subtext.

I think I like it that way. 
![image](https://user-images.githubusercontent.com/3630091/94463942-b5604800-018b-11eb-990d-4d4a5633fa67.png)

Reply:
> > I wonder how it would be with the term as the nx-list__text and the definition as the subtext.
> 
> I think I like it that way.
> ![image](https://user-images.githubusercontent.com/3630091/94463942-b5604800-018b-11eb-990d-4d4a5633fa67.png)

https://github.com/sonatype/sonatype-react-shared-components/pull/157/commits/45fe2e30628b86fdcdccae1269e4827b4d934d22

Reply:
> 45fe2e3

I would've omitted the colons, but it's not a huge deal.